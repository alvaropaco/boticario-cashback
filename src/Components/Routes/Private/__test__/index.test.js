import React from 'react';
import RoutesPrivate from '../';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom'
import { createBrowserHistory } from "history";

const history = createBrowserHistory();

const Test = () => {
    return (<>Test</>)
}

it('renderiza component RoutesPrivate', () => {
  const tree = renderer
    .create(
        <Router history={history}>
            <RoutesPrivate path="/test" component={Test} />
        </Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});