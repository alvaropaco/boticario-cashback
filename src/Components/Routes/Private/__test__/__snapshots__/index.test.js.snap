// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza component RoutesPrivate 1`] = `null`;
